experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.Handoff/BotFrameworkHttpAdapterWithHandoff.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Handoff;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Rest;
+
+namespace Microsoft.Bot.Builder.Handoff
+{
+    /// <summary>
+    /// HTTP adapter that supports handoff
+    /// When handoff is included in the SDK proper, this class becomes unnecessary.
+    /// </summary>
+    public class BotFrameworkHttpAdapterWithHandoff : BotFrameworkHttpAdapter, IHandoffAdapter
+    {
+        public BotFrameworkHttpAdapterWithHandoff(IConfiguration configuration, ILogger<BotFrameworkHttpAdapter> logger = null)
+            : base(configuration, logger)
+        {
+        }
+
+        internal BotFrameworkHttpAdapterWithHandoff()
+        {
+        }
+
+        async Task<IHandoffRequest> IHandoffAdapter.InitiateHandoffAsync(ITurnContext turnContext, Activity[] activities, object handoffContext, CancellationToken cancellationToken)
+        {
```

Comment:
Is this method public or private? I know C# defaults, but please consider being explicit.

Reply:
This is explicit interface implementation, which is always private. (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation).

This avoid exposing this method on the adapter, it can only be called from the interface.